lib/gridium.rb

```diff
@@ -35,6 +36,9 @@ def initialize
       @highlight_verifications = false
       @highlight_duration = 0.100
       @screenshot_on_failure = false
+      @screenshots_to_s3 = false
+      @project_name_for_s3 = 'gridium'
+      @subdirectory_name_for_s3 = '' #rely on GridiumS3 default
```

Comment:
we should probably put the default value for subdirectory name here, and remove it from the s3 initialize method.  These values are the default values unless the client overrides them in the Gridium.Config section of their harness.